includes/Utilities/BackgroundJobCleanup.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Utilities;
+
+use SkyVerge\WooCommerce\Facebook\Heartbeat;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class BackgroundJobCleanup
+ *
+ * Responsible for cleaning up old completed background sync jobs from SkyVerge background job system.
+ * Each job is represented by a row in wp_options table, and these can accumulate over time.
+ *
+ * Note - this is closely coupled to the SkyVerge background job system, and is essentially a patch to improve it.
+ *
+ * @see SV_WP_Background_Job_Handler
+ *
+ * @since x.x.x
+ */
+class BackgroundJobCleanup {
```

Comment:
This class name seems a bit ambiguous to me. What about something like `Facebook\Jobs\CleanupSkyvergeFrameworkJobOptions`? Since we have 3 different job systems some specificity can't hurt IMO.